src/lib/checkbox/checkbox.ts

```diff
@@ -128,11 +128,15 @@ export class MdCheckbox implements ControlValueAccessor {
 
   private _indeterminate: boolean = false;
 
+  private _color: string;
```

Comment:
Unfortunately that alone won't do the magic. The `md-checkbox` DOM element has to get the class `md-accent`. To achieve this you have the following options:

1. Add one of these lines to the constructor (All are somehow weird calls):
  - `this.color = 'accent';`
  - `this._updateColor(this._color)`
  - `this._setElementColor(this._color, true)`
  - ```this._renderer.setElementClass(this._elementRef.nativeElement, `md-${this._color}`, true);```
1. Alternative way: Modify the theme css like this (Introduce a "base" color):
```
.md-checkbox-indeterminate, .md-checkbox-checked {
  .md-checkbox-background {
    background-color: md-color($accent, 500);
  }
  &.md-primary .md-checkbox-background {
    background-color: md-color($primary, 500);
  }
  ...
}
```

Which option do you think is the best? Have a missed an option?